buildah_test.go

```diff
@@ -40,6 +40,11 @@ func TestMain(m *testing.M) {
 }
 
 func TestOpenBuilderCommonBuildOpts(t *testing.T) {
+	// This test cannot be parallized as this uses NewBuilder()
+	// which eventually and indirectly accesses a global variable
+	// defined in `go-selinux`, this must be fixed at `go-selinux`
+	// or builder must enable sometime of locking mechanism i.e if
+	// routine is creating Builder other's must wait for it.
```

Comment:
How will we know when it's safe? Is there an open issue?

Reply:
I just created an issue https://github.com/containers/buildah/issues/5967 and added in `comment` and `commit` message.